src/i18n/en.js

```diff
@@ -0,0 +1,232 @@
+// Basic constants
+const year = new Date().getFullYear();
+const urlDopplerLegacy = process.env.REACT_APP_DOPPLER_LEGACY_URL;
+const urlSite = `https://www.fromdoppler.com`;
+const urlHelp = `https://help.fromdoppler.com/en/`;
+
+// Common URLs
+const urlPrivacy = `${urlSite}/en/legal/privacy-policy`;
+const urlContact = `${urlSite}/en/contact.html`;
+const urlControlPanel = `${urlDopplerLegacy}/ControlPanel`;
+const urlBuyMonthly = `${urlControlPanel}/AccountPreferences/UpgradeAccount?Plan=monthly`;
+const urlSiteTracking = `${urlControlPanel}/CampaignsPreferences/SiteTrackingSettings`;
+
+// Markdown Links
+const linkPrivacyAndLegals_MD = `[Privacy and Legal Policy](${urlPrivacy})`;
+
+// HTML Links
+const linkSiteTracking_HTML = `<a href="${urlSiteTracking}">On-Site Tracking</a>`;
+const linkAddYourDomain_HTML = `<a href="${urlSiteTracking}">Add your domain</a>`;
+// TODO: confirm if utm parameter goes here
+const linkPrivacy_HTML = `<a target="_blank" href="${urlPrivacy}?utm_source=app&utm_medium=landing&utm_campaign=signup">Privacy Policy</a>`;
+const linkBuyMonthly_HTML = `<a href="${urlBuyMonthly}">UPGRADE NOW</a>`;
+const linkContactUs_HTML = `<a target="_blank" href="${urlContact}">contact us</a>`;
```

Comment:
I'm not sure if this is a good idea. Maybe the contents could be:

- To know more please **contact as**.

or

- For security reasons we've temporarily disabled your account. **Contact Us**.

We should have a variable for each different way of the same text. It could be result el little tedious.

Reply:
Ok, so, to clarify, I should remove `linkContactUs_HTML`, right?

Reply:
Yes, I don't know if we are going to have the same problem with the other texts. I was thinking that maybe it's not a good idea to have the HTML links in variables. But it is just only my opinion. I would like to hear other voices.

Reply:
Based on current examples, you are right, so, by the moment I will omit link constants, but we could consider that if from the content side they want to unify communication.